### ISSUE TITLE
The predict() method of StructuredDataClassifier returns class labels instead of class probabilities

### ISSUE THREAD

**Issue by matthewrenze** [keras-team/autokeras#1473]
### Bug Description
The `predict()` method of the `StructuredDataClassifier` is returning class labels instead of class probabilities. 

### Bug Reproduction
To verify, run the Structured Data Classification tutorial located on the AutoKeras website:
https://autokeras.com/tutorial/structured_data_classification/

After executing all of the code through the Data Format section, inspect the `predicted_y` variable. You will see that it contains the following data corresponding to the "Survived" class labels.

```
array([[0.],
       [0.],
       [1.],
       [1.],
       . . .
       [1.],
       [0.],
       [0.]], dtype=float32)
```

In addition, just to be 100% certain I wasn't misinterpreting the results, I replaced the existing class labels (i.e. 1's and 0's) in the Survived column with "Yes" and "No" class labels. I did this immediately after popping the survived column off of the training and test set using the following code:
```
y_train = y_train.map({0 : 'no', 1 : 'yes'})
y_test = y_test.map({0 : 'no', 1 : 'yes'})
```

The resulting `predicted_y` results were as followed: 

```
array([['no'],
       ['no'],
       ['yes'],
       . . . 
       ['no'],
       ['no'],
       ['no']], dtype='<U3')
```

So, it appears that the `predict()` method is in fact returning class labels instead of class probabilities. 

I also checked the source code for the `structured_data.py` class and the `auto_model.py` class. I cannot see anything in the code that would be explicitly trying to change the default behavior of the Keras `model.predict()` method.

### Expected Behavior
All of the documentation I've seen on Kera's says that `model.predict()` should return an array of class probabilities. However, the AutoKeras `StructuredDataClassifier.predict() `method appears to be returning class labels instead.

### Setup Details
I ran the code in GoogleColab (using the link provided in the tutorial) to eliminate any possibility of complications from my development machine:
https://colab.research.google.com/github/keras-team/autokeras/blob/master/docs/ipynb/structured_data_classification.ipynb

### Additional context
Is this the expected behavior, or am I doing something incorrectly?

If this is the expected behavior, how should we get class probabilities from our StructuredDataClassifier models in AutoKeras?

If this is not the expected behavior, would it be possible to either fix this issue or expose a method to explicitly return class probabilities vs. class labels?

Please let me know if you have any questions or require any additional information from me.

**Comment by haifeng-jin** [comment]
This is an expected behavior since we want to provide the users the end-to-end solutions.
If you want to get the probabilities, you can export the model with `automodel.export_model()`.
